pkg/reconciler/build/build_test.go

```diff
@@ -23,22 +23,28 @@ import (
 	"testing"
 	"time"
 
+	// Link in the fakes so they get injected into injection.Fake
+	fakebuildclient "github.com/knative/build/pkg/client/injection/client/fake"
+	fakebuildinformer "github.com/knative/build/pkg/client/injection/informers/build/v1alpha1/build/fake"
+	fakebtinformer "github.com/knative/build/pkg/client/injection/informers/build/v1alpha1/buildtemplate/fake"
+	fakecbtinformer "github.com/knative/build/pkg/client/injection/informers/build/v1alpha1/clusterbuildtemplate/fake"
+	fakekubeclient "github.com/knative/pkg/injection/clients/kubeclient/fake"
+	fakepodinformer "github.com/knative/pkg/injection/informers/kubeinformers/corev1/pod/fake"
+
 	"github.com/google/go-cmp/cmp"
 	v1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
-	"github.com/knative/build/pkg/client/clientset/versioned/fake"
-	informers "github.com/knative/build/pkg/client/informers/externalversions"
 	"github.com/knative/pkg/apis"
 	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	"github.com/knative/pkg/configmap"
 	"github.com/knative/pkg/controller"
-	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
 	kuberrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	kubeinformers "k8s.io/client-go/informers"
-	k8sfake "k8s.io/client-go/kubernetes/fake"
 	clientgotesting "k8s.io/client-go/testing"
 	"k8s.io/client-go/tools/cache"
+
+	. "github.com/knative/pkg/reconciler/testing"
```

Comment:
Ew, `.` imports. Can we avoid this and explicitly name things imported from `testing` ?

Reply:
Done.